src/event.js

```diff
@@ -0,0 +1,129 @@
+class Events {
+	constructor() {
+		this.eventsBox = document.getElementById('events');
+		this.eventName = document.querySelector('.eventName');
+		this.eventDate = document.querySelector('.eventDate');
+		this.eventImg = document.querySelector('.eventImg');
+		this.buttonLeft = document.querySelector(' .button-left');
+		this.buttonRight = document.querySelector(' .button-right');
+		this.counter = 0;
+		this.events = [];
+		this.mobile = window.matchMedia('min-width:576px');
+	}
+
+	showEvent() {
+		for (let e = 0; e < this.events.length; e++) {
+			e = this.counter;
+			this.eventName.innerHTML = this.events[e].name;
+			this.eventName.setAttribute('href', this.events[e].url);
+			this.eventDate.innerHTML = this.events[e].dates.start.localDate;
+			this.eventImg.src = this.events[e].images[0].url;
+			return;
+		}
+	}
+
+	moveRight() {
+		if(this.counter < this.events.length-1) {
+			this.counter += 1;
+		} else {
+			this.counter = 0;
+		}
+	}
+
+	moveLeft() {
+		if(this.counter > 0) {
+			this.counter -= 1;
+			} else {
+			this.counter = (this.events.length - 1);
+		}
+	}
+
+	showWroclawEvent() {
+		for (let e = 0; e < this.events.length; e++) {
+			e = this.counter;
+			this.eventName.innerHTML = this.events[e].offer.title;
+			this.eventName.setAttribute('href', this.events[e].offer.pageLink);
+			this.eventDate.innerHTML = this.events[e].startDate;
+			if(!this.events[e].offer.mainImage) {
+				return;
+			} else {
+				this.eventImg.src = this.events[e].offer.mainImage.standard;
+			}
+			return;
+		}
+	}
+
+	fingerMove() {
+		if(this.mobile.matches) {
+			let clientXStart = 0;
+			let clientXEnd = 0;
+			this.eventImg.addEventListener('touchstart', (e) => {
+				clientXStart = e.touches[0].clientXStart;
```

Comment:
```suggestion
				clientXStart = e.touches["0"].clientXStart;
```
z tego co widziałem to touches jest obiektem który ma dwie właściwości 0 oraz length, w taki sposób możesz dostać sie do właściwości "0"